packages/k8s/src/hooks/run-container-step.ts

```diff
@@ -108,3 +111,20 @@ function createPodSpec(
 
   return podContainer
 }
+
+function generateBuildTag(): string {
+  return `${generateRandomString()}:${generateRandomString(6)}`
+}
+
+function generateBuildHandle(): string {
+  return generateRandomString()
+}
+
+function generateRandomString(length = 10): string {
```

Comment:
any library we can leverage for generating this? what about using UUID?

Reply:
Applied :relaxed: